README.md

```diff
@@ -124,25 +124,17 @@ _Note for Windows Users: You should use a [Bash Shell][bash] to follow the steps
 
 To compile Kelp from source:
 
-1. [Download][golang-download] and [setup][golang-setup] Golang _v1.13 or later_.
-    * Confirm that `$GOPATH` is set, and that `GOBIN=$GOPATH/bin`
-    * Confirm that `$PATH` includes the full filepath of `$GOBIN`
-2. [Install Glide][glide-install] for dependency management
-    * `curl https://glide.sh/get | sh`
-3. Install [Yarn][yarn-install] and [NodeJs][nodejs-install] to build the Kelp GUI
-4. Clone the repo into `$GOPATH/src/github.com/stellar/kelp`:
-    * `git clone git@github.com:stellar/kelp.git`
-    * Change to `kelp` directory for all following commands: `cd $GOPATH/src/github.com/stellar/kelp`
-5. Install the dependencies:
-    * `glide install`
-6. Install the [astilectron-bundler][astilectron-bundler] binary into `$GOBIN`
-    * `go get -u github.com/asticode/go-astilectron-bundler/...`
+1. [Download][golang-download] and [setup][golang-setup] Golang _v1.13 or later_
+   1. Set environment variable `export GOPROXY=https://proxy.golang.org`
+2. Install [Yarn][yarn-install] and [NodeJs][nodejs-install] to build the Kelp GUI
+3. Clone the kelp repository `git clone git@github.com:stellar/kelp.git`
+4. Install the [astilectron-bundler][astilectron-bundler] binary into `$GOPATH/bin`
```

Comment:
We do not define where to set GOPATH, do we need to or can we remove it?

Reply:
I think GOPATH is synonymous with wherever your go workspace is as they will still need to  install astilectron-bundler in either where ever they set GOPATH or default $HOME/go

Reply:
I need to update the proxy value though

Reply:
Many devs are new to Golang and will get confused by the GOPATH reference here. Go modules doesn’t need GOPATH anymore so we can just suggest that they install kelp into ~/kelp and use that as the path references throughout so it is simple for the reader who is new to Golang